docs/formbuilder.md

```diff
@@ -422,6 +422,74 @@ Mit `form_exit` wird gesteuert, ob die Abarbeitung des weiteren Codes, bspw. in
 	// In PHP
 	$yform->setObjectparams('form_exit',1);
 
+### Objparams auslesen
+
+Über folgende Methode können Objparams ausgelesen werden:
+
+```php
+$yform->getObjectparams('warning')
+```
+
+### Verfügbare Objparams
+
+Folgende Objparams find verfügbar:
+- `action`: Array
+- `actions`: Aktionen des Formulars
+- `actions_executed`: Boolean. Wenn true sind die Aktionen ausgeführt. Siehe auch `preactions_executed` und `postactions_executed`.
+- `answertext`: Antworttext
+- `article_id`: Ziel Artikel ID des aFormulars
+- `clang`: Sprach ID des Formulars
+- `csrf_protection`: Boolean. Aktiviert den CSRF Schutz. Hierbei wird ein Cookie gesetzt.

```

Comment:
(bitte auf Leerzeichen in Komposita achten, es heißt CSRF-Schutz, CSS-Klasse, YForm-Template, usw.)